src/pages/app/posting_project_user/FinishCreate.js

```diff
@@ -106,29 +109,27 @@ const FinishCreate = (props) => {
           </p>
         </div>
         <div className="view-text-input-FinishCreate">
-          <input
+          <FormInput
+						formInputLabel="Email"
+						formInputPlaceholder="Your Email?" 
+						formInputValue={Email}
+						_formInputOnchangeText={setEmail}
             className="input-text-FinishCreate"
-            type="text"
-            placeholder="Email * "
-            value={Email}
-            onChange={(Email) => _onChangeEmail(Email.target.value)}
           />
-          <input
-            className="input-text-FinishCreate"
-            type="tel"
-            placeholder="Phone number"
-            value={PhoneNumber}
-            onChange={(PhoneNumber) =>
-              _onChangePhoneNumber(PhoneNumber.target.value)
-            }
-          ></input>
-          <input
-            className="input-text-FinishCreate"
-            type="text"
-            placeholder="Facebook"
-            value={Facebook}
-            onChange={(Facebook) => _onChangeFacebook(Facebook.target.value)}
-          ></input>
+
+          <FormInput
+              formInputLabel="PhoneNumber"
+              formInputPlaceholder="Your Phone number?" 
+              formInputValue={PhoneNumber}
+              _formInputOnchangeText={setPhoneNumber}
+              className="input-text-FinishCreate"
+          />
+          <FormInput
+              formInputLabel="Facebook"
+              formInputPlaceholder="Your Facebook?" 
```

Comment:
Please adjust this placeholder to:
```suggestion
              formInputPlaceholder="facebook.com/YouAreAmazing" 
```